workflow/applications/applications.py

```diff
@@ -210,3 +223,44 @@ def is_monotonic(test_list: List, check_decreasing: bool = False) -> bool:
             return all(x > y for x, y in zip(test_list, test_list[1:]))
         else:
             return all(x < y for x, y in zip(test_list, test_list[1:]))
+
+    def check_globus(self, conf):
+        # This method checks that globus can be used on this platform
+        # and is configured properly.
+
+        # Test that globus can be imported
+        spec = importlib.util.find_spec("globus_cli")
+        if spec is None:
+            raise ImportError("Globus-cli module not found!  Check that the module is loaded!")
+
+        globus_conf = conf.parse_config(['config.base', 'config.globus'])
+
+        # Initialize globus
+        globus = which("globus")
+
+        if globus is None:
+            raise FileNotFoundError("Could not find the globus command!")
+
+        # Check that a globus connection to the server is open
+        globus_output = globus("session", "show", output=str).splitlines()[2:]
+
+        local_uid_found = False
+        rdhpcs_uid_found = False
+
+        # There should be two sessions (MSU and RDHPCS), but if someone is running
+        # this elsewhere (e.g. NOAA cloud), it may be just one (RDHPCS).
+        local_uid = os.environ['LOGNAME'].lower()
+        for line in globus_output:
+            uid = line.split("|")[0].split("@")[0].lower()
+            domain = line.split("|")[0].split("@")[1].lower()
+
+            if uid == local_uid:
+                local_uid_found = True
+            if "rdhpcs" in domain:
+                rdhpcs_uid_found = True
+
+        if not local_uid_found or not rdhpcs_uid_found:
+            print(f"ERROR a globus session is not yet established on {globus_conf.SERVER_NAME}")
+            print("      Please establish a globus connection!")
```

Comment:
use logger, and should this `raise` an exception?

Reply:
Agreed on both fronts.  I'll move the `exception` from line 122 here.

Reply:
Done and tested.